tests/GravityFormsConnectWiseTest.php

```diff
@@ -80,6 +80,13 @@ function test_addon_settings_should_have_fields() {
 		$this->assertTrue( array_key_exists( "save_callback", $actual[0]["fields"][3] ) );
 		$this->assertTrue( array_key_exists( "feedback_callback", $actual[0]["fields"][3] ) );
 
+		$this->assertEquals( $actual[0]["fields"][4]["name"], "client_id" );
+		$this->assertEquals( $actual[0]["fields"][4]["label"], "Client ID" );
+		$this->assertEquals( $actual[0]["fields"][4]["type"], "text" );
+		$this->assertEquals( $actual[0]["fields"][4]["class"], "small" );
```

Comment:
I just saw sometimes we use single quote and sometimes double quote. Which one that we really use?

Reply:
It uses one single quote 
but I  change related with code before